notifications/defaultPreferences.ts

```diff
@@ -1,18 +1,26 @@
-import { NotificationPreferences } from '../graphql/types/preferences';
+type EmailChannel = { email: boolean };
+type Channel = { [channel: string]: boolean };
+export type Preference = { [category: string]: EmailChannel };
 
-export const DEFAULT_PREFERENCES: NotificationPreferences = {
+export const DEFAULT_PREFERENCES: Preference = {
```

Comment:
I guess the default prefs for marketing notifications should be "off" so `false`.